frontend/src/constants/meta.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * The default metadata for the site. Can be extended and/or overwritten per page. And even in components!
+ * See the Nuxt.js docs for more info.
+ * {@link https://nuxtjs.org/guides/features/meta-tags-seo} Nuxt.js Docs
+ */
```

Comment:
I think this doc comment was associated with the `meta` variable on L10.

Reply:
Right, good catch! Thank you.